jsapp/js/components/permissions/sharingForm.scss

```diff
@@ -138,6 +139,25 @@ $s-gray-row-spacing: 10px;
   }
 }
 
+.form-modal__item--permissions-header {
+  font-weight: bold;
+}
+
+.form-modal__item--anonymous-submissions {
+  display: flex;
+  padding-bottom: sizes.$x12;
+  padding-top: sizes.$x8;
+  border-bottom: 1px solid colors.$kobo-gray-96;
+
+  .toggle-switch {
+    padding-right: sizes.$x8;
+  }
+
+  .checkbox__label {
+    max-width: 100% !important;
+  }
```

Comment:
Not sure if this needs to be removed or re-factored, but it might've been missed in the checkbox -> toggle refactor.